### ISSUE TITLE
Padding in window.

### ISSUE THREAD

**Issue by sophuric** [Authenticator-Extension/Authenticator#390]
**Describe the bug:** There is padding in the window, this might be the scrollbar.

![image](https://user-images.githubusercontent.com/48314599/65728942-e08b0400-e0ff-11e9-8a75-948d76f87606.png)

Also when I scroll down, the menubar goes, and there is even more of a gap at the bottom of the window.

![image](https://user-images.githubusercontent.com/48314599/65728932-d8cb5f80-e0ff-11e9-9284-8ba147f1a5a2.png)

**Platform:** 
 - Browser: Chrome
 - Browser Version: 77.0.3865.90 (Official Build) (64-bit)

**Comment by Sneezry** [comment]
Did you scale the popup page? You can find scale settings from menu->popup page->scale

**Comment by sophuric** [comment]
@Sneezry Yes, I have it at 125% which is the problem. So can you please fix it. At all the other scales this padding doesn't appear.

**Comment by Sneezry** [comment]
We use CSS `scale` to scale the UI roughly. However, it worked really well in the past. I have no idea if it's a CSS bug on Chrome. I will try to fix this issue in the next a few days.

**Comment by dhulwells** [comment]
Can I take a crack at this?

**Comment by mymindstorm** [comment]
@dhulwells go right ahead! Relevant file is sass/popup.scss

**Comment by mymindstorm** [comment]
and this function

https://github.com/Authenticator-Extension/Authenticator/blob/7b9d9527d29f01bb13e7758c1daa370081997d31/src/store/Menu.ts#L43-L49

**Comment by sophuric** [comment]
I've changed the `select` tag values and the gap starts to appear at 112%.
![Animation showing that the side padding appears at 112%](https://user-images.githubusercontent.com/48314599/67131253-a1a82400-f24f-11e9-89fa-42c32d0cfd16.gif)
